### PR TITLE
Add pushy@1.0.0 w/ git auto-update config

### LINKED ISSUE #8301: [Request] Pushy.js
**Library name:** Pushy.JS
**Git repository url:**https://github.com/christophery/pushy
**npm package url(optional):** 
**License(s):**The MIT License (MIT)
**Official homepage:**https://www.christopheryee.ca/pushy/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
PR for #8301

@pvnr0082t 
Please help me review it, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required):https://github.com/christophery/pushy
- Official website (optional, not the repository):https://www.christopheryee.ca/pushy/
- NPM package url (optional):
- GitHub / Bitbucket popularity (required):
  - Count of stars:1154
  - Count of watchers:61
  - Count of forks:191
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

close #8301, cc @christophery

### CODE REVIEW

File: ajax/libs/pushy/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "url": "https://github.com/christophery/pushy.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/christophery/pushy.git",
+    "basePath": "",
+    "files": [
+      "css/pushy*",
+      "js/pushy*"
```
Comment (pvnr0082t) [inline ajax/libs/pushy/package.json]: You can use `"+(css|js)/pushy*"` here.

File: ajax/libs/pushy/1.0.0/css/pushy.css [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+}
+.pushy-submenu > a::after {
+  content: '';
+  display: block;
+  height: 11px;
+  width: 8px;
+  position: absolute;
+  top: 50%;
+  right: 15px;
+  background: url("../img/arrow.svg") no-repeat;
```
Comment (pvnr0082t) [inline ajax/libs/pushy/1.0.0/css/pushy.css]: @x09326 I think file in `img` folder also need to be added.

**Comment by x09326** [comment]
@pvnr0082t 
I have modified the file map in **package.json**.
And I have added img file and minified ```pushy.css```.
Please help me review it, thanks.

File: ajax/libs/pushy/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "url": "https://github.com/christophery/pushy.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/christophery/pushy.git",
+    "basePath": "",
+    "files": [
+      "+(css|js)/pushy*",
+      "img/arrow.svg"
```
Comment (pvnr0082t) [inline ajax/libs/pushy/package.json]: @x09326 I think "img/*.+(svg|gif|png)" will be more flexible.

**Comment by x09326** [comment]
@pvnr0082t 
done!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@x09326 I suggest you to reply on the place you got the comment/request for change.

File: ajax/libs/pushy/package.json (hunk 3 above)
Comment (PeterDaveHello) [inline ajax/libs/pushy/package.json]: If you'll like to use this schem, please also add `jpg` extension it that. Personally I prefer use a more flexible for img like `img/**/*`, images are usually not so messy as js/css files do.

**Comment by x09326** [comment]
@pvnr0082t 
I have modified the file map to ```img/**/*```.
Please help me review it, thanks.

**Comment by pvnr0082t** [comment]
LGTM

File: ajax/libs/pushy/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/pushy/package.json]: @x09326 @pvnr0082t ...

**Comment by PeterDaveHello** [comment]
@x09326 @pvnr0082t as you can see if you don't reply on the origin review place you'll miss it.

File: ajax/libs/pushy/package.json (hunk 1 above)
Comment (pvnr0082t) [inline ajax/libs/pushy/package.json]: oh...I see now.

File: ajax/libs/pushy/package.json (hunk 3 above)
Comment (x09326) [inline ajax/libs/pushy/package.json]: I have modified it.

**Comment by x09326** [comment]
ping @pvnr0082t @PeterDaveHello 
I have rebased with the latest master br.
Please help me review it, thanks.

**Comment by x09326** [comment]
@pvnr0082t  @PeterDaveHello 
The ci test has finished.
Please help me review it, thanks.

**Comment by kennynaoh** [comment]
LGTM

**Pull request merged**